src/cli/lint/annotate/github/eslint.ts

```diff
@@ -0,0 +1,29 @@
+import * as GitHub from '../../../../api/github';
+import { ESLintOutput } from '../../../../cli/adapter/eslint';
+
+export const createEslintAnnotations = (eslint: ESLintOutput) => {
+  const annotations: GitHub.Annotation[] = [];
+  [...(!eslint.ok ? eslint.errors : []), ...eslint.warnings].forEach(
+    (result) => {
+      result.messages.forEach((message) => {
+        // Annotations only support start_column and end_column on the same line.
+        const column = message.line === message.endLine && message.column;
+        const endColumn =
+          message.line === message.endLine && (message.endColumn || column);
+
+        annotations.push({
+          annotation_level: message.severity === 2 ? 'failure' : 'warning',
+          start_line: message.line || 1,
+          end_line: message.endLine || message.line || 1,
```

Comment:
Are we trying to handle empty strings here, or can we use nullish coalescing `??`?

Reply:
Yeah just handling `undefined`. I've switched it to the nullish coercing operator. I noticed while doing some manual testing that eslint parsing errors don't give us a start_line, so in order for the file to show up in the `Files Changed` tab we have to give it Line 1.

eg.

![image](https://user-images.githubusercontent.com/18017094/139575102-ad7b5401-31eb-47dd-b846-b475154b1664.png)